client/disputes/evidence/index.js

```diff
@@ -178,7 +206,11 @@ export const DisputeEvidenceForm = ( props ) => {
 			<Card size="large" key={ section.key }>
 				<CardHeader>{ section.title }</CardHeader>
 				<CardBody>
-					{ section.description && <p>{ section.description }</p> }
+					{ readOnly
```

Comment:
This can be simplified as:

```javascript
{ readOnly && section.description && ( <p>{ section.description }</p> ) }
```

Reply:
I'm not a JS master, but I don't think suggested shortcut follows the same logic: show `section.description` if it is available and if it is not in the `readOnly` mode.

Reply:
@dmvrtx Good catch! I missed this in a hurry. It should be:

```javascript
{ ! readOnly && section.description && ( <p>{ section.description }</p> ) }
```

Reply:
```
{ ! readOnly && section.description && ( <p>{ section.description }</p>) }
```
I think this works! Updated in the code.